data/core/units/orcs/Sorcerer.cfg

```diff
@@ -0,0 +1,102 @@
+#textdomain wesnoth-units
+
+[unit_type]
+    id=Orcish Sorcerer
+    name=_"Orcish Sorcerer"
```

Comment:
Since you changed it to `gender=female`..I think this should be now "Orcish Sorceress"